docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,80 @@
 ---
 title: Tinylicious
 menuPosition: 1
-status: unwritten
+editor: sdeshpande3
 ---
 
 ## What is Tinylicious?
 
-Our local service used for prototyping
-
-Link to repo
+Tinylicious is a local, in-memory Fluid service intended for prototyping and development purpose. The [TinyliciousClient]({{< relref "tinyliciousclient.md" >}}) is designed to work specifically with the Tinylicious service.
 
 ## Using Tinylicious locally
 
+You can run `Tinylicious` locally by executing the following command:
+
+```sh
+npx tinylicious@latest
+```
+
+By default, `Tinylicious` runs on port `7070`. You can change port by setting the `PORT` environment variable when running Tinylicious. Setting environment variables will vary based on the shell you are using. For example, the `Windows PowerShell` commands below will run Tinylicious on port `6502`.
```

Comment:
```suggestion
By default, Tinylicious runs on port 7070. You can change port by setting the `PORT` environment variable when running Tinylicious. Setting environment variables will vary based on the shell you are using. For example, the Windows PowerShell commands below will run Tinylicious on port 6502.
```

Remove all the code formatting for words like Tinylicious and Windows PowerShell. Same for the port numbers, but since `PORT` is referring to the variable, leave the code formatting.